helix-term/src/commands/typed.rs

```diff
@@ -2008,7 +1995,7 @@ pub static TYPABLE_COMMAND_MAP: Lazy<HashMap<&'static str, &'static TypableComma
             .collect()
     });
 
-pub fn command_mode(cx: &mut Context) {
+pub(super) fn command_mode(cx: &mut Context) {
```

Comment:
Why the change here?

Reply:
Before my making `write_all_impl` pub, everything else exposed for import in commands.rs was done so through `pub(super)`. I assumed this was supposed to as well and was trying to maintain consistency.